README.md

```diff
@@ -350,7 +352,7 @@ Services to securely store your Docker images.
 * [Haven](https://github.com/codeabovelab/haven-platform) - Haven is a simplified container management platform that integrates container, application, cluster, image, and registry managements. By [@codeabovelab](https://github.com/codeabovelab)
 * [Helios](https://github.com/spotify/helios) - A simple platform for deploying and managing containers across an entire fleet of servers by [@spotify](spotify)
 * [Kontena](https://github.com/kontena/kontena) - Application Containers for Masses https://www.kontena.io/
-* [Kubernetes][kubernetes] - Open source orchestration system for Docker containers by Google [kubernetes] See Also [awesome-kubernetes](https://github.com/ramitsurana/awesome-kubernetes) by [@ramitsurana][ramitsurana]
+* [Kubernetes][https://github.com/kubernetes/kubernetes] - Open source orchestration system for Docker containers by Google
```

Comment:
Oh, just found a wrong Markdown formatting. Link should be within parentheses like:

`[Kubernetes](https://github.com/kubernetes/kubernetes)`

Please fix that as well.